magenta-lib/src/main/scala/magenta/deployment_type/DeploymentType.scala

```diff
@@ -53,7 +53,7 @@ case class Param[T](name: String,
 
   def get(pkg: DeploymentPackage)(implicit reads: Reads[T]): Option[T] =
     pkg.pkgSpecificData.get(name).flatMap(jsValue => Json.fromJson[T](jsValue).asOpt)
-  def apply(pkg: DeploymentPackage, reporter: DeployReporter)(implicit reads: Reads[T], manifest: Manifest[T]): T = {
+  def apply(pkg: DeploymentPackage, reporter: DeployReporter)(implicit reads: Reads[T]): T = {
```

Comment:
Sneaking the removal of this unused implicit into this (noticed it whilst looking up reflection)